lib/galaxy/webapps/galaxy/services/invocations.py

```diff
@@ -252,6 +254,26 @@ def write_store(
         rval = async_task_summary(result)
         return rval
 
+    def report_error(
+        self, trans: ProvidesUserContext, invocation_id: DecodedDatabaseIdField, payload: ReportInvocationErrorPayload
+    ) -> ReportInvocationErrorResponse:
+        # ensure_celery_tasks_enabled(trans.app.config) # TODO: Should this be enabled?
+        workflow_invocation = self._workflows_manager.get_invocation(
+            trans, invocation_id, eager=True, check_ownership=False, check_accessible=True
+        )
+        email = payload.email
+        if not email and not trans.anonymous:
+            email = trans.user.email
+        messages = trans.app.error_reports.default_error_plugin.submit_invocation_report(
+            invocation=workflow_invocation,
+            user_submission=True,
+            user=trans.user,
+            email=email,
+            message=payload.message,
+            trans=trans,
+        )
+        return ReportInvocationErrorResponse(messages=messages)
```

Comment:
What is that needed for ? I think a 200 status code is enough here

Reply:
Don't respond with anything, what's the point of returning the messages ? You can even do 204